This patch was written in-house and is intended for contribution upstream.

--- tmux-2.2/osdep-sunos.c.orig	2015-04-25 13:44:57.000000000 -0700
+++ tmux-2.2/osdep-sunos.c	2016-08-25 08:56:09.437920629 -0700
@@ -18,6 +18,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/param.h>
 
 #include <event.h>
 #include <fcntl.h>
